gpuflow/dataplane/gpu/dataplane_lpm_gpu.cpp

```diff
@@ -47,7 +47,42 @@ int DataPlaneLPMIPv6GPU::CreateLPMTable() {
     std::cerr << "Create lpm table error from ipv6 lpm factory";  
     exit(1);
   }
-  // TODO: Add ipv6 lpm rules here.
+
+  // Add the rule_neighbor_solicitation: Multicst the Neighbor Solicitation packet.
+  uint8_t ipv6_addr_ns[16];
+  ipv6_addr_ns[0] = 255;
+  ipv6_addr_ns[1] = 2;
+  ipv6_addr_ns[2] = 0;
+  ipv6_lpm_factory.AddLPMRule(ipv6_addr_ns, 24, 255);
+  
+  // Add the rule_0: Direct the 2220:0000::/24 packets to port 0. 
+  uint8_t ipv6_addr_0[16];
+  ipv6_addr_0[0] = 34;
+  ipv6_addr_0[1] = 32;
+  ipv6_addr_0[2] = 0;
+  ipv6_lpm_factory.AddLPMRule(ipv6_addr_0, 24, 0);
+
+  // Add the rule_1: Direct the 2220:0100::/24 packets to port 1.
+  uint8_t ipv6_addr_1[16];
+  ipv6_addr_1[0] = 34;
+  ipv6_addr_1[1] = 32;
+  ipv6_addr_1[2] = 1;
+  ipv6_lpm_factory.AddLPMRule(ipv6_addr_1, 24, 1);
+
+  // Add the rule_2: Direct the 2220:0200::/24 packets to port 2.
+  uint8_t ipv6_addr_2[16];
+  ipv6_addr_2[0] = 34;
+  ipv6_addr_2[1] = 32;
+  ipv6_addr_2[2] = 2;
+  ipv6_lpm_factory.AddLPMRule(ipv6_addr_2, 24, 2);
+
+  // Add the rule_3: Direct the 2220:0300::/24 packets to port 3.
+  uint8_t ipv6_addr_3[16];
+  ipv6_addr_3[0] = 34;
+  ipv6_addr_3[1] = 32;
+  ipv6_addr_3[2] = 3;
+  ipv6_lpm_factory.AddLPMRule(ipv6_addr_3, 24, 3);
+
```

Comment:
Use loop to reduce redundant codes.
```c++
for (uint8_t i = 0; i < 4; i++) {
  // Ensure explicit initialization.
  uint8_t ipv6_net_addr[16] = {0};
  ipv6_net_addr[0] = 34;
  ipv6_net_addr[1] = 32;
  ipv6_net_addr[2] = i;
  ipv6_lpm_factory.AddLPMRule(ipv6_net_addr, 24, i);
}
```

Reply:
Advice: eliminate the variable names containing meaningless numbers. Such as `addr_0, addr_1`.